REST/AddFeaturesOnTimer.py

```diff
@@ -0,0 +1,40 @@
+import threading
+import arcpy  
+import requests
+import json
+from datetime import datetime
+
+
+# Disable warnings
+requests.packages.urllib3.disable_warnings()
+
+# Get Parameters
+addfsURL = arcpy.GetParameterAsText(0)
+interval = int(arcpy.GetParameterAsText(1))
+deviceID = int(arcpy.GetParameterAsText(2))
+quantity = int(arcpy.GetParameterAsText(3))
+username = arcpy.GetParameterAsText(4)
+password = arcpy.GetParameterAsText(5)
```

Comment:
Can we remove ArcPy to make this pure REST? Can just hardcode the parameters for now, something like:
```python
addfsURL = 'url'
interval = 2
...
```

Reply:
done - I just used sys instead.